File: shoppet_Backend/src/main/java/devkb/studio/shoppet/model/payment.java
package devkb.studio.shoppet.model;

import lombok.Builder;
import lombok.Data;

import java.sql.Date;

@Builder
@Data
public class payment {
    private String payment_id;
    private String order_id;
    private String payment_method;
    private boolean status;
    private Date created_at;
}

File: shoppet_Fontend/lib/Model/apiModel/payment.dart
import 'dart:ffi';

class payment {
  final String payment_id;
  final String order_id;
  final String payment_method;
  final bool status;
  final String created_at;

  payment({required this.payment_id, required this.order_id, required this.payment_method, required this.status, required this.created_at});

  factory payment.fromJson(Map<String, dynamic> json) {
    return payment(
      payment_id: json['payment_id'] as String,
      order_id: json['order_id'] as String,
      payment_method: json['payment_method'] as String,
      status: json['status'],
      created_at: json['created_at'] as String,
    );
  }

}